composer.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "pod-point/reviews-php",
+  "description": "A review service for PHP.",
+  "type": "library",
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "Pod Point Software Team",
+      "email": "software@pod-point.com"
+    }
+  ],
+  "require": {
+    "php": ">=7.1",
+    "guzzlehttp/guzzle": "~5.3|~6.0",
+    "illuminate/support": "5.*",
+    "laravel/framework": "5.*"
```

Comment:
we don't need this a dependency do we?

Reply:
Kinda, we need it because we use the Laravel classes. Like the ServiceProvider class and the Laravel config helpers. But if there's a better way of telling the consumer package that we need this, Then I'm happy to change it to a another one or two imports.

Reply:
yes please I think that would be better

Reply:
Ok, so I tried this and ended just going down a rabbit of dependencies. One of the problems is that we need the helper functions that are only available in the "laravel/framework" package.

Let me know what you think?